### PR TITLE
Fix some English grammar

### PR DESCRIPTION
**Pull request by dotellie**
I am still studying Japanese, but I can certainly help out with English! I fixed some small grammatical errors in the README.
日本語はまだ勉強中ですが、英語なら手伝いますよ！ READMEの文法をちょっとだけ直しました。

My intention is to help out more with development from here on (also with programming). Therefore, if you have anything you need help with, please tell me!
これからも開発を手伝いつもりですから（プログラミングもできます）、何か手伝ってほしいことがあるなら是非、私に言ってください！

よろしくおねがいします！

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,19 +8,19 @@
...
 
-**This branch is still under development for next version (V2).**
+**This branch is still in development for the next version (V2).**
 
 ### Contributing
 
 #### Requirements
 
-To run Annict on local machine, you need to install some software below:
+To run Annict on a local machine, you need to have the following dependencies installed:
```
Comment (wopian) [inline README.md:17]: This can just be `you need to install the following dependencies:`

Comment (dotellie) [reply README.md:17]: True, although I didn't want to change the original text too much. I was struggling with what to do with this line for a while though. 😅

Comment (shimbaco) [reply README.md:17]: @wopian Thank you for your support! It's good in simple! 😄

**Comment by shimbaco** [comment]
@magnonellie That’s very helpful and informative! Thanks a lot! 日本語とても上手ですね！ :satisfied:

**Pull request merged**